extended/src/main/java/io/kubernetes/client/extended/generic/GenericKubernetesApi.java

```diff
@@ -332,7 +332,7 @@ public GenericKubernetesApi(
               listOptions.getLimit(),
               listOptions.getResourceVersion(),
               listOptions.getTimeoutSeconds(),
-              null,
+              false,
```

Comment:
this pull also fixes watch api for GenericKubernetesApi which is uncovered by more detailed testing..